modules/planning/tasks/deciders/path_reference_decider/path_reference_decider.cc

```diff
@@ -91,6 +91,12 @@ Status PathReferenceDecider::Process(Frame *frame,
       frame->learning_based_data().learning_data_adc_future_trajectory_points();
   ADEBUG << "There are " << path_reference.size() << " path points.";
 
+  // when learning model has no output, use rule-based model instead.
+  if (path_reference.size() == 0) {
+    ADEBUG << "No learning model output, using rule-based planning instead";
+    return Status::OK();
+  }
+
```

Comment:
total_path_counter_ shall be moved ahead of this.
(1) it's OK not to count frames skip this decider on purpose (side pass, lane change and etc). other than that, they shall be counted to total.
(2) or to be more accurate, count all frames. so that we know the exact ratio of valid path reference. 
(3) or provide two ratio.  valid/all_frames,   valid /frames_calling_model

Reply:
Fixed in the next commit. Thank you.